packages/style-loader/README.md

```diff
@@ -83,32 +79,11 @@ module.exports = {
   use: ['@neutrinojs/style-loader']
 };
 
-// Usage showing default options
+// Usage with options
 module.exports = {
   use: [
     ['@neutrinojs/style-loader', {
-      style: {},
-      css: {
-        importLoaders: opts.loaders ? opts.loaders.length : 0
-      },
-      loaders: [],
-      test: /\.css$/,
-      ruleId: 'style',
-      styleUseId: 'style',
-      cssUseId: 'css',
-      modules: true,
-      modulesSuffix: '-modules',
-      modulesTest: /\.module.css$/,
-      extractId: 'extract',
-      extract: {
-        enabled: process.env.NODE_ENV === 'production',
-        loader: {},
-        plugin: {
-          filename: process.env.NODE_ENV === 'production'
-            ? 'assets/[name].[contenthash:8].css'
-            : 'assets/[name].css'
-        }
-      }
+      test: /\.scss$/,
```

Comment:
What do you think about replacing this example with `modules: false` or `extract: false` etc, since the changing of `test` here might suggest that's all that is required for supporting SCSS? (And it also overlaps with the section below)